frontend/src/tutorial/TutorialTooltip.tsx

```diff
@@ -19,14 +20,24 @@ const TutorialTooltip: React.FC<TooltipRenderProps> = ({
     backProps,
     size,
     isLastStep,
+    closeProps,
 }) => {
     return (
         <Card
             data-cy='tutorial-tooltip'
             sx={{ maxWidth: 'md', width: { xs: '95vw', sm: undefined } }}
             {...tooltipProps}
         >
-            <CardHeader title={step.title} />
+            <CardHeader
+                title={step.title}
+                action={
+                    <IconButton arra-label='close' onClick={closeProps.onClick}>
+                        <Typography component='span' color='Gray'>
```

Comment:
Removing the typography component here and just passing CloseIcon will probably fix the weird shape. You can pass a color directly to the close icon. Also, can we put gray in lowercase?

Reply:
Sadly, the color prop type does not accept this value - unless my memory fails me, hence the typography. However, the weird shape (if you're talking about the one on hover) is there regardless. I will try again with these regards though, if I can't fix it with the methods you advise, what then?